chain/vm/vm.go

```diff
@@ -43,9 +43,11 @@ import (
 
 const MaxCallDepth = 4096
 
-var log = logging.Logger("vm")
-var actorLog = logging.Logger("actors")
-var gasOnActorExec = newGasCharge("OnActorExec", 0, 0)
+var (
+	log            = logging.Logger("vm")
+	actorLog       = logging.Logger("actors")
+	gasOnActorExec = newGasCharge("OnActorExec", 0, 0)
+)
```

Comment:
Ideally we would introduce formatting changes in separate PRs, so that future git blames can be less noisy and more accurate. But this is OK.

Reply:
I'm happy to revert the formatting if you prefer

Reply:
No worries.